concourse/pipelines/pg_regress_pipeline.yml

```
@@ -55,46 +55,46 @@ resources:
     repository: gcr.io/data-gpdb-ud/gpdb-pxf-dev/gpdb6-centos7-test-pxf-hdp2
     tag: latest
     username: _json_key
-    password: ((pxf-cloudbuild-service-account-key))
+    password: ((ud/pxf/secrets/pxf-cloudbuild-service-account-key))
 
 # ~/workspace/gp-continuous-integration/secrets/gpdb6-integration-testing.dev.yml
 - name: libquicklz-centos7
   type: gcs
   source:
-    bucket: ((gcs-bucket))
+    bucket: ((ud/pxf/common/gcs-bucket))
```

```suggestion
    bucket: ((ud/pxf/common/gpdb-concourse-resources))
```

I think the name `gcs-bucket` is too vague.